.github/workflows/run-tests-monitor.yml

```
@@ -3,7 +3,7 @@ name: Monitor Tests
 on:
   push:
     branches:
-      - main
+    - main
```

Help, we lost proper indentation again
```suggestion
      - main
```

---

thanks, man! :beer: